extensions/sql-migration/src/wizard/targetSelectionPage.ts

```diff
@@ -133,9 +137,14 @@ export class TargetSelectionPage extends MigrationWizardPage {
 					break;
 				}
 				case MigrationTargetType.SQLVM: {
-					if (!this.migrationStateModel._targetServerInstance ||
-						resourceDropdownValue === constants.NO_VIRTUAL_MACHINE_FOUND) {
+					let targetVm = this.migrationStateModel._targetServerInstance as SqlVMServer;
```

Comment:
unless targetVm needs to be reassigned, I suggest using 'const' over 'let' by default.